src/services/configServices/NetlifyTomlService.js

```diff
@@ -1,3 +1,4 @@
+const { Base64 } = require("js-base64")
```

Comment:
Hmm this one also :(

Reply:
This one is needed! For some reason it wasn't being imported before